download.tt

```diff
@@ -0,0 +1,339 @@
+        for an overview of how to install and use Nix.
+      </li>
+      <li>Old releases are <a href="/releases/nix">also available</a>.</li>
+    </ul>
+  </section>
+</div>
+
+<script charset="utf-8">
+  $('#getting-nix a').click(function (e) {
+    e.preventDefault();
+    $(this).tab('show');
+  })
+
+</script>
+
+
+<h1>Getting NixOS</h1>
+
+<p class="lead">
+  Below you will find installation instructions for NixOS, the linux
+  distribution. The latest stable release series is
+  <strong>[%latestNixOSSeries%]</strong>
+</p>
+
+<ul id="getting-nixos" class="nav nav-tabs">
+  <li class="active"><a href="#nixos-iso">ISO image</a></li>
+  <li><a href="#nixos-virtualbox">VirtualBox</a></li>
+  <li><a href="#nixos-amazon">Amazon EC2</a></li>
+  <li><a href="#nixos-azure">Microsoft Azure</a></li>
+  <li><a href="#nixos-google">Google Compute Engine</a></li>
+  <li><a href="#nixos-more">More ...</a></li>
+</ul>
+
+<div class="tab-content">
+  <section class="tab-pane active" id="nixos-iso">
+    <p>You can install NixOS on physical hardware by burning one of the CD
+      images onto a blank CD/DVD disk, or <a href="https://nixos.org/nixos/manual/index.html#sec-booting-from-usb">by
+        copying it onto a USB stick</a>. For <strong>installation
+        instructions</strong>, please see the <a href="[%nixosManual%]">manual</a>.</p>
+
+    <div class="alert alert-block">
+      Please note that NixOS at the moment lacks a nice, user-friendly
+      graphical installer. Therefore this form of installation may not be
+      suitable for novice Linux users.</div>
+
+    <p>The <strong>graphical installation CD</strong> contains the NixOS
+      installer as well as X11, Plasma 5 Desktop and several applications. It’s a
+      <em>live CD</em>, so it allows you to get an impression of NixOS (and
+      the Nix package manager) before installing it.</p>
+
+    [% prefix = "https://channels.nixos.org/nixos-" _ latestNixOSSeries %]
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-graphical-x86_64-linux.iso">Graphical live CD, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-graphical-x86_64-linux.iso.sha256">SHA-256</a>)
+        <span class="label label-info">Recommended for most users</span></li>
+    </ul>
+
+    <p>The <strong>minimal installation CD</strong> does not contain X11,
+      and is therefore a lot smaller. You have to run the installer from
+      the console. It contains a number of rescue tools.</p>
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-minimal-x86_64-linux.iso">Minimal installation CD, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-minimal-x86_64-linux.iso.sha256">SHA-256</a>)</li>
+      <li><a href="[%prefix%]/latest-nixos-minimal-i686-linux.iso">Minimal installation CD, 32-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-minimal-i686-linux.iso.sha256">SHA-256</a>)</li>
```

Comment:
The SHA-256 hash is linked, but it is not explained what it is, why it is useful and how to check the image with it

I suggest to add an **info** box under it with this details and commands to check the image file

Reply:
Lets improve this in next PR.